go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/hashicorp/consul
 
 go 1.20
```

Comment:
We could also bump the version here
```suggestion
go 1.22
```

Reply:
I'll try it and make sure all of our tests still pass.